src/main.tsx

```diff
@@ -1,13 +1,47 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
-import App from './App.tsx';
 import { Provider } from 'react-redux';
 import { store } from './store';
+import { createBrowserRouter, RouterProvider } from 'react-router-dom';
+import Home from './pages/Home/Home';
+import Community from './pages/Community/Community';
+import Chat from './pages/Chat/Chat';
+import Profile from './pages/Profile/Profile';
+import Login from './pages/Login/Login';
+import SignUp from './pages/SignUp/SignUp';
```

Comment:
**This is awful**, because it is very cluttered and **has to change**. 

You have a few options here:
If you want to stick with default exports (keep in mind if you do then we use them everywhere, its an all or nothing thing):
- make an index.ts file inside src/pages, export all the defaults from the pages and name them "export { default as Home } from './Home/Home'" for example. Then in this file you can instead do import {Home, Community, etc... } from './pages'
- OR your 2nd option is to do named exports and export * from each page which would then allow you to do the same thing
- either option is fine but leaving it like this is not

Reply:
In commit 3a55dea4e3775d7ba5687d6ce2a78152fa43df46, I added this "barrel export" method. Please review it again to ensure it's configured the way you like :blush: